lib/specdiff/config.rb

```diff
@@ -9,8 +9,20 @@ class << self
     attr_reader :config
   end
 
-  DEFAULT = Config.new(colorize: true).freeze
-  @config = DEFAULT.dup
+  # Generates the default configuration
+  def self.default_configuration
+    config = Config.new(colorize: true)
+
+    if !ENV["NO_COLOR"].nil? && !ENV["NO_COLOR"].empty?
```

Comment:
consider a spec for this (in `specdiff_spec.rb`)

it might be unexpected that NO_COLOR=0 sets color to `true`